e2e-tests/playwright/e2e/plugins/rbac/rbac.spec.ts

```diff
@@ -150,6 +151,42 @@ test.describe.serial("Test RBAC plugin as an admin user", () => {
     await uiHelper.verifyCellsInTable(allCellsIdentifier);
   });
 
+  test("Should download the user list", async () => {
+    await page.locator('a:has-text("Download User List")').click();
+    const fileContent = await downloadAndReadFile(page);
+    const lines = fileContent.trim().split("\n");
+
+    const header = "userEntityRef,displayName,email,lastAuthTime";
```

Comment:
nitpick

can you please extract header validation to a specific function, for example:
```
function validateFileHeader(actualHeader: string, expectedHeader: string): void {
  if (actualHeader !== expectedHeader) {
    throw new Error(`Header mismatch: expected "${expectedHeader}" but found "${actualHeader}"`);
  }
```

and them you can call :

```
const lines = fileContent.trim().split("\n");
const expectedHeader = "userEntityRef,displayName,email,lastAuthTime";
validateFileHeader(lines[0], expectedHeader);
```